doc/tutorials/highlevel-bindings.md

```diff
@@ -54,6 +54,29 @@ sufficient to write (or generate) a one-to-one mirror of the C API and use that
 The generated code should still be understandable. So if writing a more idiomatic API on top of the
 direct mapping, significantly simplifies the generated code (and maybe also the template), you should write such an API.
 
+### Bindings for the `lowlevel` Library
+
+If you decide on mapping the functionality of kdb.h 1:1 it is pretty straight-forward.
+
+Your programming language of choice must provide a way to call into C code, e.g.: 
+* [cgo](https://golang.org/cmd/cgo/) for the Go programming language
+* [jni](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/) for JAVA
+* [n-api](https://nodejs.org/api/n-api.html) for javascript
```

Comment:
I do not think these examples are relevant. And n-api is a bad example.

Reply:
I will remove them